src/util.h

```diff
@@ -415,10 +415,12 @@ class CLicenceDlg : public CBaseDlg
     CLicenceDlg ( QWidget* parent = nullptr );
 
 protected:
+    QCheckBox*   chbAgree;
```

Comment:
Trying to understand why `chbAgree` has been moved into the class rather than being local to the constructor, as it was previously?

Reply:
Because I _had_ been using it outside the constructor...  I thought I'd reverted all that change.